### PR TITLE
Increase rootfs size to 270Mb

### PR DESCRIPTION
**Pull request by rucoder**
- We have only ~2Mb left in roofs. It is time to increase its size

### CODE REVIEW

**Review by rene — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -364,8 +364,8 @@ ifeq ($(LINUXKIT_PKG_TARGET),push)
 endif
 
 # Though the partition size is set to 512MB lets check for ROOTFS_MAXSIZE_MB not exceeding 450MB for kubevirt.
-# That seems to be the direction taken for existing kvm systems where partition size is 300MB but
-# rootfs size is limited to 250MB. That helps in catching image size increases earlier than at later stage.
+# That seems to be the direction taken for existing kvm systems where partition size is 512MB but
```
Comment (eriknordmark) [inline Makefile:367]: 512 are the new (since release TBD) but all the older use 300. The comments need to reflect this.

File: Makefile [hunk 2]
```diff
@@ -364,8 +364,8 @@ ifeq ($(LINUXKIT_PKG_TARGET),push)
...
+# rootfs size is limited to 350MB. That helps in catching image size increases earlier than at later stage.
```
Comment (eriknordmark) [inline Makefile:368]: Can't do that when 99% of what's deployed has a partition size of 300MB.

How big is the debug container? Can we handle it the same way we handle k3s and longhorn in EVE kubevirt i.e., remove it from the rootfs?

**Review by eriknordmark — CHANGES_REQUESTED** [review]
Must allow updates of deployed EVE-OS with a partition size of 300MB. So that is the abolute max.

**Comment by rucoder** [comment]
> Must allow updates of deployed EVE-OS with a partition size of 300MB. So that is the abolute max.

ah, yes. I knew that 250 limit in not that soft for regular EVE. I'll change to 300. @rene so the size of the image is still the issue and eve cannot grow beyond 300Mb in any near futur. I'm afraid VIM is still on my to-remove list :)

**Comment by christoph-zededa** [comment]
> > Must allow updates of deployed EVE-OS with a partition size of 300MB. So that is the abolute max.
> 
> ah, yes. I knew that 250 limit in not that soft for regular EVE. I'll change to 300. @rene so the size of the image is still the issue and eve cannot grow beyond 300Mb in any near futur. I'm afraid VIM is still on my to-remove list :)

Why not remove `perf`?

**Review by OhmSpectator — APPROVED** [review]

**Comment by OhmSpectator** [comment]
Looks fine to me, but we need @eriknordmark to approve (he has requested changes).

**Review by eriknordmark — APPROVED** [review]
LGTM but please create an issue to indicate that we need to get back to 250 again (even if that takes a while and might require to explore different approaches such as separating out some service containers like debug and edge-view, or figuring out which packages can be removed from various containers, or ...)

**Comment by OhmSpectator** [comment]
> LGTM but please create an issue to indicate that we need to get back to 250 again (even if that takes a while and might require to explore different approaches such as separating out some service containers like debug and edge-view, or figuring out which packages can be removed from various containers, or ...)

https://github.com/lf-edge/eve/issues/4358

**Pull request merged**